RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -2238,7 +2238,7 @@
 				MACOSX_DEPLOYMENT_TARGET = 10.13;
 				MTL_ENABLE_DEBUG_INFO = INCLUDE_SOURCE;
 				MTL_FAST_MATH = YES;
-				PRODUCT_BUNDLE_IDENTIFIER = com.revenuecat.Purchases;
+				PRODUCT_BUNDLE_IDENTIFIER = com.revenuecat.purchaseTester;
```

Comment:
This should be `com.revenuecat.Purchases` I believe, it's the bundle ID from the framework itself
<img width="759" alt="image" src="https://user-images.githubusercontent.com/3922667/151435581-6e70c4ae-c16b-4886-80cd-f85908c42cc7.png">

Reply:
Doh! I don't know why I changed this lol... I think I changed something when I was in the wrong Xcode project 😛